### ISSUE TITLE
polls: preview seems wrong, when I change my vote, but do not submit

### ISSUE THREAD

**Issue by fuzzylogic2000** [liqd/a4-meinberlin#3867]
**URL:** 
**user:** 
**expected behaviour:** 
**behaviour:** when the vote is changed, but not submitted, the preview shows the not submitted vote as answer, while the results stay the same
**important screensize:**
**device & browser:** 
**Comment/Question:** 

Screenshot?
![Screenshot from 2021-09-06 13-51-07](https://user-images.githubusercontent.com/8178179/133222400-7b466d1c-e0e5-4989-8b54-20c98b9edb2a.png)

**Comment by fuzzylogic2000** [comment]
discussion so far:
@mcastro-lqd and @fuzzylogic2000 talked about it and thought, that the preview button should either be disabled when there are achnges that are not yet submitted or to have a warning that the results are not yet submitted when clicking that (but with the latter being rather complicated)

@CarolingerSeilchenspringer: I am not sure if I got it right. As I can see now the vote is only counted after hitting "Antwort senden", although the little check in the preview changes before I hit that button. Is that the problem? I would expect that my answer only counts after hitting the button (again). Can't we make the check have the same behaviour, so only change after hitting the button?  And about the dialoge: I think it would be better to not show the preview instead of browser alerts. Does that help? 
@fuzzylogic2000: yes! If we don't show the preview link before sending, we don't have the problem with the weird behaviour, because then there is no way of getting there, right? Or do we also want a cancel-button then? So, if I changed or added votes, I have the "send" and the "cancel" button and only if these are both inactive (because I didn't change anything or hit cancel), the preview link is seen?

**Comment by mcastro-lqd** [comment]
I have left the options on Zeplin as well to make it clearer: https://zpl.io/bLoDDdJ

**Comment by fuzzylogic2000** [comment]
We came up with a four-part solution:

1. Everything on the result page does not change until the vote is actually submitted
2. When I already voted and come back to the page, I see the result screen
3. When I already voted, at the end of the result screen, the link says "Antwort ändern" instead of "Zur Umfrage"
4. When I already voted, the submit button on the vote screen says "Antwort ändern" instead of "Antwort abgeben"

Anything to add @CarolingerSeilchenspringer @mcastro-lqd @khamui ?
